runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -9806,6 +9806,36 @@ static TR::Register *inlineSinglePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT_FATAL(node->getNumChildren() == 3, "Wrong number of children in inlineFPTrg1Src3");
```

Comment:
Can this assert be more helpful - can it state the number of children expected and the number of children found as well as the node number?

Reply:
Fixed in 2f9a77